examples/deployment/docker/log_signer/Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM golang:1.9
 
-ENV DB_USER=test \
-    DB_PASSWORD=zaphod \
-    DB_DATABASE=test \
-    DB_HOST=127.0.0.0:3306
+ENV DB_FLAG="--mysql_url=test:zaphod@tcp(127.0.0.1:3306)/test" \
+    DB_PROVIDER="mysql"
 
 ENV HOST=0.0.0.0 \
     HTTP_PORT=8091
 
 ENV SEQUENCER_GUARD_WINDOW=0s \
     FORCE_MASTER=true \
-    SEQUENCER_INTERVAL=10s \
+    SEQUENCER_INTERVAL=300ms \
     NUM_SEQ_FLAG=10 \
-    BATCH_SIZE=50
+    BATCH_SIZE=2000
```

Comment:
And a big increase. These are based on our settings?

Reply:
50 is tiny :)
2000 is what I was running it with while I was writing it up.

Reply:
Yep, again this might be a bit high for SQL.